src/main/java/lotto/domain/WinningNumbers.java

```diff
@@ -0,0 +1,67 @@
+package lotto.domain;
+
+import lotto.common.ErrorCode;
+import lotto.common.WinningType;
+
+import java.util.Arrays;
+import java.util.HashSet;
+import java.util.Set;
+
+public class WinningNumbers {
+    private static final String winningNumbersTextSplitRegex = ",";
```

Comment:
상수이므로 대문자 표기 부탁 드립니다 😄 
```suggestion
    private static final String WINNING_NUMBERS_TEXT_SPLIT_REGEX= ",";
```